### PR TITLE
Python2 support and other improvements

### PR DESCRIPTION
**Pull request by aravindavk**
- Python 2 support
- Added wait till etcd cluster is ready
- Removed retry noise in output

Signed-off-by: Aravinda VK <avishwan@redhat.com>

### CODE REVIEW

**Comment by aravindavk** [comment]
@kotreshhr @amarts ^^

**Comment by aravindavk** [comment]
> Any particular reason for changing defaultClusterSize?

To support minikube. Testing with minikube is not complete, facing some issue with device add.

File: kubectl_gluster/main.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 import yaml
 
 
-defaultClusterSize = 3
+defaultClusterSize = 1
```
Comment (amarts) [inline kubectl_gluster/main.py]: https://github.com/aravindavk/kubectl-gluster/blob/ff3b519aa1cd087dc7704cee558eeec2e01ef6a2/kubectl_gluster/main.py#L46 is where it matters. With this change, it won't work on 3 node cluster. We need to remove/handle cluster size check also then.

Comment (aravindavk) [reply kubectl_gluster/main.py]: got it. Fixed now.

**Pull request merged**